### ISSUE TITLE
Problem:About list.   TypeError: 'type' object is not subscriptable.

### ISSUE THREAD

**Issue by Chris-Lee-2028** [alex-87/qaekwy-python#3]
I got this error when I ran the example program since my python version is 3.8. I guess this error is about **_list_**
TypeError: 'type' object is not subscriptable
![Error](https://github.com/user-attachments/assets/f6b08081-5645-4bbe-8627-9e167d15d95a)

**Comment by alex-87** [comment]
Yes, indeed, this is related to the version of Python. The [prerequisites](https://docs.qaekwy.io/docs/installation/#prerequisites) indicates that Python `3.8` is supported, however not any of the [tags](https://github.com/alex-87/qaekwy-python/tags) (_up to [v0.1.4](https://github.com/alex-87/qaekwy-python/releases/tag/v0.1.4)_) embedded the fix that has been Merged from https://github.com/alex-87/qaekwy-python/pull/2. This issue will be fixed by tagging + [releasing](https://pypi.org/project/qaekwy/) what has been done at [qaekwy.response#L281](https://github.com/alex-87/qaekwy-python/blob/main/qaekwy/response.py#L281).

However, I highly encourage to **upgrade Python version** to a [supported version](https://devguide.python.org/versions/).
